src/main/cliapp/src/service/useGetUserSettings.js

```diff
@@ -10,7 +10,7 @@ export const useGetUserSettings = (key, defaultValue) => {
 	const [settings, setSettings] = useState(() => {
 		const stickyValue = localStorage.getItem(key);
 		const stickyObject = stickyValue !== null ? JSON.parse(stickyValue) : defaultValue;
-		if(key !== 'themeSettings') {
+		if(key !== 'themeSettings' && key !== 'DiseaseAnnotationsFormSettings') {
```

Comment:
Maybe we should pass down a boolean like `isTable` and default it to false and then in places where this method is being used for a table, pass it in as true. Otherwise, this if check is just going to keep growing.

Reply:
Sure Adam, that's a good approach. Will modify as mentioned.